tensorflow_lite_support/ios/BUILD

```diff
@@ -179,3 +184,24 @@ ios_static_framework(
         "//tensorflow_lite_support/ios/task/text/qa:TFLBertQuestionAnswerer",
     ],
 )
+
+ios_static_framework(
+    name = "TensorFlowLiteTaskAudio_framework",
+    hdrs = [
+        ":TFLCommon.h",
+        ":TFLBaseOptions.h",
+        ":TFLClassificationOptions.h",
+        ":TFLCategory.h",
+        ":TFLClassificationResult.h",
+        ":TFLAudioFormat.h",
+        ":TFLFloatBuffer.h",
+        ":TFLAudioRecord.h",
+        ":TFLAudioTensor.h",
+        ":TFLAudioClassifier.h",
+    ],
+    bundle_name = "TensorFlowLiteTaskAudio",
+    minimum_os_version = "10.0",
```

Comment:
Can this be 9.0 (TFL_MINIMUM_OS_VERSION) rather than 10.0?

If it needs to be 10.0, could you define this as a constant (e.g. TFL_TASK_MINIMUM_OS_VERSION) to share with the vision pod?

Reply:
According to our discussion, I have made the deployment target for audio consistent with vision (10.0) and retained text at 9.0